app/views/boats/index.html.erb

```diff
@@ -7,18 +7,23 @@
         <%= link_to "Add a new boat", new_boat_path, class: "btn btn-default btn-lg" %>
       </div>
     </div>
-
-    <div class="container">
+<div class="container">
       <div class="row">
         <% @boats.each do |boat| %>
-          <div class="col-md-4">
+          <div class="col-md-3">
```

Comment:
De 4e kolom schoof eronder, in het midden. Dit was een lelijk gezicht dus
heb ik het zo even opgelost.
De css klopt niet helemaal dus dit kunnen we later veranderen.

Er moet ook een counter komen die checkt hoeveel kolommen er in 1 'row' div
zijn. Als de 'row' vol is, moet er een nieuwe 'row' gegenereerd worden
etc...


Met vriendelijke groet,
Kind regards,

Iris Bune

2016-02-10 10:43 GMT+01:00 navase <notifications@github.com>:

> In app/views/boats/index.html.erb
> <https://github.com/navase/boatconnection/pull/14#discussion_r52434698>:
>
> >        <div class="row">
> >          <% @boats.each do |boat| %>
> > -          <div class="col-md-4">
> > +          <div class="col-md-3">
>
> Ik vind 3 kolommen mooier dan 4, maar misschien komt dat ook doordat ik
> hem niet fullscreen bekijk. Moeten we het met de css nog over hebben denk
> ik.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/navase/boatconnection/pull/14/files#r52434698>.
>